### DISCUSSION TITLE
append  td not work

### DISCUSSION THREAD

**Discussion by enhengenheng-hubowen** [GrapesJS/grapesjs#4460] [Q&A]
component.getChildAt(0).getChildAt(0).append(`<td style=" border: 1px dashed;height:20px">text</td>`)

![image](https://user-images.githubusercontent.com/52563642/179942902-d0628f93-8377-4130-8af5-4f8e0c3b99ce.png)

**Comment by artf** [comment]
Yeah unfortunately here the native DomParser is cleaning up the passed HTML string. As a workaround, you can pass directly the component definition and skip the parser.
```js
component.append({ 
 type: 'cell', 
 attributes: { style: 'border: 1px dashed; height: 20px' }, 
 components: 'Text' 
})
```
